Tests/MeiliSearchUnitTests/SettingsTests.swift

```diff
@@ -42,11 +40,21 @@ class SettingsTests: XCTestCase {
 
     // MARK: Settings
 
+    func testShouldInstantiateFromEmptyData() {
+        let stubSetting = buildStubSetting(from: "{}")
+
+        XCTAssertTrue(stubSetting.rankingRules.isEmpty)
+        XCTAssertEqual(stubSetting.searchableAttributes, ["*"])
+        XCTAssertEqual(stubSetting.displayedAttributes, ["*"])
+        XCTAssertTrue(stubSetting.stopWords.isEmpty)
+        XCTAssertTrue(stubSetting.synonyms.isEmpty)
+    }
+
     func testGetSetting() {
 
         //Prepare the mock server
 
-        let stubSetting: Setting = buildStubSetting()
+        let stubSetting = buildStubSetting(from: json)
```

Comment:
Why is the typing removed ?

Reply:
@bidoubiwa Yes, we would like to keep the types explicit.

Reply:
No issue, what is the reason behind this requirement?

I tend to remove any duplication wherever I can, less code = less headaches but I can revert this if needed.

Reply:
This is needed to enforce that types are the same as expected (question yourself: what's the return type of the function `buildStubSetting` without the need of going to the function?), also it helps Xcode to index the code faster, we suffer for constant problems where Xcode is not capable of indexing it faster than a human. :upside_down_face:

Reply:
Ok, I never encountered those kind of issues before, even on far bigger projects (which mixed ObjC and Swift) but I will follow your coding guidelines.

As for the naming of the function, it should be called getStubSetting to say that it actually returns this 😉

Reply:
Fixed in `0eeac0c`.